tf_quant_finance/black_scholes/vanilla_prices.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 """Black Scholes prices of a batch of European options."""
```

Comment:
These changes are not related to American option pricing. Could you please separate out this change and send it. It looks good, so should not take much time to go through the review?